darwin/importer/formats/labelbox.py

```diff
@@ -85,7 +87,7 @@ def parse_file(path: Path, validate: Callable[[Any], None]) -> Optional[List[Ann
 
     with path.open() as f:
         data = json.load(f)
-        validate(data)
+        validate(data, labelbox_export)
```

Comment:
for consistency's sake
```suggestion
        validate(data, schema=labelbox_export)
```